### PR TITLE
Edited the Column class in order to give the possibility to inject custom Render object in the Grid definitions.

### PR DESCRIPTION
**Pull request by dop3**
Edited the Column class in order to give the possibility to inject custom Render object in the Grid definitions.

Eg: $this->addColumn("Username")
	 ->setRenderType("AppBundle\Grid\Render\Link")
	 ...;
This will use the given class as a Render.

This will give the users of this bundle the possibility to create and use custom Render.

### CODE REVIEW

**Comment by dop3** [comment]
I've added another change in order to have two different twig function to render separately the grid's html and the grid's js.

**Comment by pedro-teixeira** [comment]
Nice changes @dop3, I'll check them as soon as possible!

**Comment by pedro-teixeira** [comment]
Can you add some documentation at https://github.com/pedro-teixeira/grid-bundle/blob/master/Resources/doc/column/filter.md#filter-types ?

File: Twig/GridExtension.php [hunk 1]
```diff
@@ -162,4 +176,42 @@ public function renderGrid(GridView $gridView)
             );
         }
     }
+
+    /**
+     * Render (only html) grid view
+     *
+     * @param GridView $gridView
+     *
+     * @return mixed
```
Comment (pedro-teixeira) [inline Twig/GridExtension.php]: The return here is `string`

File: Twig/GridExtension.php [hunk 2]
```diff
@@ -162,4 +176,42 @@ public function renderGrid(GridView $gridView)
...
+            );
+        }
+    }
+
+    /**
+     * Render (only js) grid view
+     *
+     * @param GridView $gridView
+     *
+     * @return mixed
```
Comment (pedro-teixeira) [inline Twig/GridExtension.php]: Same here

Comment (dop3) [inline Twig/GridExtension.php]: Hi Pedro.
I Don't think so (or I'm missing some).
This method returns null or what the inner "renderBlock" calls returns. And as per your annotations, the "renderBlock" method returns "mixed".
Am I right?
Thanks...

Comment (pedro-teixeira) [inline Twig/GridExtension.php]: Ok, that's true

**Comment by dop3** [comment]
I'm going to add info on the documentation about custom FILTER and custom RENDER.

**Comment by dop3** [comment]
I've edited the docs adding info on rendering grid's html and js separately

**Pull request merged**